client/src/redux/createStore.js

```diff
@@ -29,10 +29,18 @@ const composeEnhancers = composeWithDevTools({
 });
 
 export const createStore = () => {
-  const store = reduxCreateStore(
-    rootReducer,
-    composeEnhancers(applyMiddleware(sagaMiddleware, epicMiddleware))
-  );
+  let store;
+  if (process.env.FREECODECAMP_NODE_ENV === 'production') {
```

Comment:
```suggestion
  if (ENVIRONMENT === 'production') {
```
`process` is not available in the browser, but `ENVIRONMENT` is defined globally.

Could you also add `/* global ENVIRONMENT */` at the start of the file?  Otherwise it will fail during linting.

Reply:
I think this will have to go thru our config setup for env variables using the `env.json`, I can take a look later this week, unless someone beats me to it.

Reply:
I think this *should* work, but avoiding globals entirely seems like a good plan.